app/controllers/application_controller.rb

```diff
@@ -34,4 +34,11 @@ def squash_multiparameter_datetime_attributes(params, attribute_names)
   def notify_bad_request(_exception)
     render plain: "Error: One or more recipients not in GOV.UK Notify team (code: 400)", status: :bad_request
   end
+
+  def require_govuk_editor(redirect_path: root_path)
+    return if current_user.govuk_editor?
+
+    flash[:danger] = "You do not have permission to see this page."
+    redirect_to redirect_path
```

Comment:
I think it's good to have the flexibility right from the beginning, especially because we already know at this point that we'll need it later on.
This way we avoid an unnecessary change in a later commit.